app/src/test/java/in/testpress/testpress/util/ValidationsTest.java

```diff
@@ -0,0 +1,34 @@
+package in.testpress.testpress.util;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+
+@RunWith(PowerMockRunner.class)
+public class ValidationsTest {
+
+    @Test
+    public void test_phone_number_validation() {
```

Comment:
MethodName_ExpectedBehavior_StateUnderTest

Reply:
Changed test name according to the convention asked.